internal/function/commit_stats.go

```diff
@@ -96,20 +99,39 @@ func (f *CommitStats) Eval(ctx *sql.Context, row sql.Row) (interface{}, error) {
 
 	r, err := f.resolveRepo(ctx, row)
 	if err != nil {
-		return nil, err
+		ctx.Warn(0, "commit_stats: unable to resolve repository")
+		logrus.WithField("err", err).Error("commit_stats: unable to resolve repository")
+		return nil, nil
 	}
 
+	log := logrus.WithField("repository", r)
+
 	to, err := f.resolveCommit(ctx, r, row, f.To)
 	if err != nil {
-		return nil, err
+		ctx.Warn(0, "commit_stats: unable to resolve 'to' commit of repository: %v", r)
+		log.WithField("err", err).Error("commit_stats: unable to resolve 'to' commit")
+		return nil, nil
 	}
 
 	from, err := f.resolveCommit(ctx, r, row, f.From)
 	if err != nil {
-		return nil, err
+		ctx.Warn(0, "commit_stats: unable to resolve 'from' commit of repository: %v", r)
+		log.WithField("err", err).Error("commit_stats: unable to resolve from commit")
+		return nil, nil
 	}
 
-	return commitstats.Calculate(r.Repository, from, to)
+	result, err := commitstats.Calculate(r.Repository, from, to)
```

Comment:
This will fail for root commits too, when commit.Parent(0) fails.

Reply:
Yep, that's why the error is not returned